api.swagger.yaml

```diff
@@ -6395,6 +6564,472 @@ components:
+              type: object
+              properties:
+                uuid:
+                  description: 'Invotra UUID of the user.'
+                  type: string
+                  format: uuid
+                  example: '01234567-89ab-cdef-1234-56789abcdef0'
+            team:
+              description: 'The team targeting this custom content.'
+              type: object
+              properties:
+                uuid:
+                  description: 'Invotra UUID of the team.'
+                  type: string
+                  format: uuid
+                  example: '01234567-89ab-cdef-1234-56789abcdef0'
+            locations:
+              description: 'The locations targeting this custom content.'
+              type: array
+              items:
+                type: object
+                properties:
+                  uuid:
+                    description: 'Invotra UUID of the location.'
+                    type: string
+                    format: uuid
+                    example: '01234567-89ab-cdef-1234-56789abcdef0'
+            grades:
+              description: 'The grades targeting this custom content.'
+              type: array
+              items:
+                type: object
+                properties:
+                  uuid:
+                    description: 'Invotra UUID of the grade.'
+                    type: string
+                    format: uuid
+                    example: '01234567-89ab-cdef-1234-56789abcdef0'
+            regions:
+              description: 'The regions targeting this custom content.'
+              type: array
+              items:
+                type: object
+                properties:
+                  uuid:
+                    description: 'Invotra UUID of the region.'
+                    type: string
+                    format: uuid
+                    example: '01234567-89ab-cdef-1234-56789abcdef0'
+          required:
+            - custom_content_type
+            - title
+            - body
+            - state
+            - site_section
+            - review_date
+            - owner
+    custom-content-schema-common:
+      type: object
+      properties:
+        custom_content_type:
+          description: 'The the custom content type machine name.'
+          type: string
+          example: 'guideline'
+        title:
+          description: 'Title of the custom content.'
+          type: string
+          example: 'Guidelines for creating a strong password'
+        body:
+          description: 'The body of the custom content.'
+          type: string
+          example: 'You can create strong passwords using an online random password generator.'
+        state:
+          description: 'Indicates the workflow state of the custom content.'
+          type: string
+          enum:
+            - 'draft'
+            - 'needs review'
+            - 'reviewed'
+            - 'published'
+            - 'unpublished'
+        schedule_date_timestamp:
```

Comment:
We're going to have to take this out of the common schema, as you shouldn't be able to schedule content during creation, only when updating (and reading back)

Reply:
taken out of POST